openmetadata-service/src/main/java/org/openmetadata/service/search/elasticsearch/ElasticSearchClient.java

```diff
@@ -2607,4 +2607,46 @@ public void removeILMFromComponentTemplate(String componentTemplateName) throws
           "Failed to remove ILM policy from component template: " + e.getMessage());
     }
   }
+
+  @SuppressWarnings("unchecked")
+  public Map<String, Object> clusterStats() throws IOException {
```

Comment:
use ClusterStatsRequest and ClusterStatsResponse here

Reply:
we cannot, it will be speicific to implementation and it requires to be uniform from Elastic and OpenSearch